core/templates/services/UpgradedServices.ts

```diff
@@ -17,11 +17,11 @@
  */
 
 import { downgradeInjectable } from '@angular/upgrade/static';
+import { HttpClient, HttpXhrBackend } from '@angular/common/http';
```

Comment:
Just to check, will the changes in this file affect other active PRs related to angular migration? If so, we will need to ask the authors to sync with develop once your PR is merged.

Reply:
It's an old change right now. :P